hie-plugin-api/Haskell/Ide/Engine/Cradle.hs

```diff
@@ -201,7 +201,10 @@ Just (Ex (ProjLocV2File { plProjectDirV2 = "/Foo/"}))
 
 In the given example, it is not guaranteed which project type is found,
 it is only guaranteed that it will not identify the project
-as a cabal v1-project.
+as a cabal v1-project. Note that with latest cabal-helper, by default a *.cabal file
```

Comment:
Should we maybe give a specific version of cabal-helper here? Either by commit hash, or tag, whatever. Or even/also just put the date in brackets. These comments tend to hang around forever.

Reply:
Fair, good idea.